project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.module

```diff
@@ -55,3 +55,26 @@ function c4m_content_photo_c4m_helper_entity_label_info() {
     ),
   );
 }
+
+/**
+ * Implements hook_preprocess().
+ *
+ * Create the back to photo album link.
+ */
+function c4m_content_photo_preprocess(&$variables, $hook) {
+  if ($hook != 'node_gallery_api_item_navigator') {
+    return;
+  }
+
+  global $base_path;
+  $options = array(
+    'absolute' => TRUE,
+  );
+
+  // 'gallery_link' contain the full path to the gallery (back button). This create duplication of the base path in
```

Comment:
@nirgn975 
Is it just me this one is gonna fail on the sniffer for a line that is longer than 80 chars ?
:\